publish2cloud.py

```diff
@@ -270,26 +273,37 @@ def publish_to_remote_settings(config, section, chunknum):
 
     list_name = config.get(section, 'output')
     chunk_file = chunk_metadata(open(config.get(section, 'output'), 'rb'))
+
+    # Default data
     record_data = {
         'id': list_name,
         'Categories': categories,
         'ExcludedCategories': excluded_categories,
         'Type': list_type,
         'Name': list_name,
         'Checksum': chunk_file['checksum'],
-        'Version': chunknum,
+        'Version': chunknum
     }
+
+    # Add fields for versioned lists
+    if version is not None:
+        record_data['id'] = f'{list_name}-{math.trunc(version.release[0])}'
+
+        next_version = version.release[0] + 1
+        record_data['filter_expression'] = f'env.version|versionCompare("{version}") >= 0 && env.version|versionCompare("{next_version}") < 0'
```

Comment:
It would be better to add a comment here to explain how the `filter_expression` works for versioning.